src/main/java/com/googleintern/wfm/ruleengine/model/FilterModel.java

```diff
@@ -0,0 +1,25 @@
+package src.main.java.com.googleintern.wfm.ruleengine.model;
+
+import com.google.auto.value.AutoValue;
+
```

Comment:
Add javadoc for public class. 
Same for everywhere else.

https://google.github.io/styleguide/javaguide.html#s7.1.1-javadoc-multi-line